scripts/sql/mods/mod_1_new_job_pool_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE `pooled_sts_job` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
can we change the table name to sts_job_pool

Reply:
implemented